src/api/gitHub.js

```diff
@@ -277,9 +207,10 @@ export const storeRepoMetrics = async (organization) => {
   spinner.succeed(`Exporting Completed: ${path}`);
 };
 
+
 /**
- * Determine if the user is targeting a GHES instance or not.
- *
+ * Determine if the user is targeting a GHES instance or not. 
+ * 
  * * @param {[Objects]} server the graphql endpoint for a GHES instance
  */
 export const determineGraphQLEndpoint = async (server) => {
```

Comment:
This doesn't need to be `async` because it doesn't use promises anywhere. You can remove this, and then avoid the `await` when you call the function:

```suggestion
export const determineGraphQLEndpoint = (server) => {
```

Reply:
I changed the logic as suggested.